runtime/src/tokio/runtime.rs

```diff
@@ -279,6 +280,8 @@ impl crate::Runner for Runner {
                 let config = IoUringNetworkConfig {
                     tcp_nodelay: self.cfg.network_cfg.tcp_nodelay,
                     iouring_config: iouring::Config {
+                        // TODO (#1045): make `IOURING_NETWORK_SIZE` configurable
+                        size: IOURING_NETWORK_SIZE,
```

Comment:
In `tokio_connectivity`, we have 10 peers (all listening) that all may have multiple messages inflight. This gets very close to the 128 default size limit that iouring defaults to.

This increases the default size for iouring networking to a sane value (although it should probably be isolated between listening and sending).